src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -76,6 +60,21 @@
   static final int SUGGESTED_DECIMAL_PLACES_FOR_VOTE_ARITHMETIC = 4;
   static final BigDecimal SUGGESTED_MINIMUM_VOTE_THRESHOLD = BigDecimal.ZERO;
   static final int SUGGESTED_MAX_SKIPPED_RANKS_ALLOWED = 1;
+  private static final int MIN_COLUMN_INDEX = 1;
```

Comment:
I hate this change and fought tooth and nail against the code reorganization during commit to make sure it didn't happen. Just ended up unchecking it in the commit dialog.

Reply:
Hmm. I don't want to have to remember not to let the code reorganization run every time I edit this file. Can we just live with the way that it wants to do it?

Reply:
Ok ok, just make sure the comment about false default booleans is in a place that makes sense. There's a small chance that me changing this to a String might resolve the issue, but won't know until I work on it more on Sunday.